src/main/java/models/data/Project.java

```diff
@@ -0,0 +1,26 @@
+package models.data;
+
+public class Project implements IProject {
+    private String description;
+    private String members;
+
+    public Project(String description, String members) {
+        this.description = description;
+        this.members = members;
```

Comment:
Why is members a string? Is this just a placeholder so we can create a class for members later on?

Reply:
placeholder because Member class not created